src/Tiqr/Legacy/TiqrUserRepository.php

```diff
@@ -67,4 +70,16 @@ public function getUser(string $userId): TiqrUser
 
         return new TiqrUser($this->userStorage, $this->userSecretStorage, $userId);
     }
+
+    public function userStorageHealthCheck(string &$message = ''): bool
```

Comment:
Suggestion. Do not include the message in our own health check port. But simply log if there was a message. Do not bubble it into our own implementation?